websites/P/Pixelmon Mod/presence.ts

```diff
@@ -0,0 +1,64 @@
+const presence = new Presence({
+		clientId: "976435781486911509"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const page = document.location.pathname,
+		presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		},
+		title = document.querySelector(
+			"#page-body > main > div > div.col-md-9 > div.side-segment > h3 > a"
+		),
+		title2 = document.querySelector(
+			"#page-body > main > div > div.col-md-9 > div.side-segment > h3"
+		),
```

Comment:
Could you use [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) here instead?